### PR TITLE
LPS-159264 | Audit and refactor tests for SegmentationEditSegment.testcase

### PR DESCRIPTION
**Pull request by rafaellajordao**
JIRA Ticket:
[LPS-159249](https://issues.liferay.com/browse/LPS-159249)
↳ [LPS-159264](https://issues.liferay.com/browse/LPS-159264)
Motivation:
Audit and refactor tests for SegmentationEditSegment.testcase

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase [hunk 1]
```diff
@@ -86,6 +86,11 @@ definition {
 			Segmentation.viewSegment(
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase:86]: Hey @rafaellajordao ! I think it's missing to remove the Segment.viewSegment macro from the tests.

**Comment by rafaellajordao** [comment]
> ###  ci:test:tango - 171 out of 182 jobs passed in 1 hour 47 minutes
> Click here for more details.

these failures have nothing to do with the test automation :)

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase (hunk 1 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationEditSegment.testcase]: yeah! i will fix :+1:

**Review by Luizcarlosqueiroz — APPROVED** [review]
Good to go! :smile:

**Comment by Luizcarlosqueiroz** [comment]
ci:forward

**Pull request closed without merging**